homeassistant/components/analytics/analytics.py

```diff
@@ -115,6 +123,43 @@ async def save_preferences(self, preferences: dict) -> None:
                 self.hass, self.preferences.get(ATTR_DIAGNOSTICS, False)
             )
 
+    async def _gather_configured_domains(self) -> Set:
```

Comment:
Can you explain what you're trying to achieve here that we can't get from `hass.config.components` ? Everything that is in that set is set up.

Reply:
`hass.config.components` includes integrations that are only dependencies of other integrations